core/modules/filter/filter.pages.inc

```diff
@@ -296,8 +296,17 @@ function filter_format_editor_image_form($form, &$form_state, $format) {
   );
 
   // Incorporate the Library View in the editor dialog form.
+  $view_name = 'image_library';
+  $view = views_embed_view($view_name);
+  if (empty($view)) {
+    $view = t('The view "@view" is either not available.', array('@view' => $view_name));
+    if (user_access('administer views')) {
+      $view .= ' ' . t('Check the <a href="!url">Views administration page</a> to enable or create the view as needed.', array('!url' => url('admin/structure/views')));
```

Comment:
Can we say which view? 
Change `...to enable or create the view as needed.` to `...to enable or create an Image Library view.`